### DISCUSSION TITLE
Not have to re-render unchanged documents

### DISCUSSION THREAD

**Discussion by rosikand** [quarto-dev/quarto-cli#2774] [Q&A] [answered]
I have a big directory for a listing page which consists of over 50 posts. Whenever I make the slightest change or add a new document, i have to re-render everything to update my site by running `quarto render` from the root with the specified output directory in `_quarto.yml`. Is there anyway to only render the changed page so i don't have to wait until all other pages also render that have been previously rendered?

**Comment by mcanouil** [comment]
Did you look at the `freeze` option?
>You can use the freeze option to denote that computational documents should never be re-rendered during a global project render, or alternatively only be re-rendered when their source file changes:

Source: <https://quarto.org/docs/projects/code-execution.html#freeze>

**Comment by mcanouil** [comment, marked as answer]
FYI:
- https://github.com/quarto-dev/quarto-cli/discussions/3674
